### PR TITLE
feat(ci): add publish of package in test pypi and pypi for tags

### LINKED ISSUE #17: [ci] add GitHub action workflow for package publication
Create a new .github/workflows/package-publish.yml to add package publication to CI.

- [x] define pypi user that will be used for the publication
- [x] add secrets in GitHub settings to store pypi user and password
- [x] define github action used for pypi publication

### PR DESCRIPTION
**Pull request by jmkerloch**
closes #17 

update setup.py because an invalid classifier was used.

### CODE REVIEW

**Comment by lgrd** [comment]
There is a need to change the README.md. Because some links are broken inside https://test.pypi.org/project/r2gg/. Wa could use complete links to github. What do you think of it ?

**Comment by jmkerloch** [comment]
> There is a need to change the README.md. Because some links are broken inside https://test.pypi.org/project/r2gg/. We could use complete links to github. What do you think of it ?

Yes we could use link to the static documentation webpage.

**Comment by Guts** [comment]
I find it a bit strange that the maintainer identified on PyPi is a user who doesn't correspond to anything: neither Github organisation, nor legal person, nor physical person.

**Comment by lgrd** [comment]
> I find it a bit strange that the maintainer identified on PyPi is a user who doesn't correspond to anything: neither Github organisation, nor legal person, nor physical person.

As I said, it is temporary. ;) The goal is to move on the IGN organization, when it will be available.

**Comment by Guts** [comment]
> > I find it a bit strange that the maintainer identified on PyPi is a user who doesn't correspond to anything: neither Github organisation, nor legal person, nor physical person.
> 
> As I said, it is temporary. ;) The goal is to move on the IGN organization, when it will be available.

Sorry, I was not aware of this saying.

**Review by lgrd — CHANGES_REQUESTED** [review]

File: .github/workflows/pypi_publish.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
```
[the remark this answers is not in the document]
Comment (jmkerloch) [reply .github/workflows/pypi_publish.yml:3]: Unfortunately we can't publish the same version number, not on `test.pypi` and not on `pypi`. 

The step with `test.pypi` is mainly there to check that there is no issue with package build.

And for the publication on `pypi` it's only done when a tag is used: https://github.com/IGNF/route-graph-generator/pull/30/files#diff-e63c5d93f83e5bdcd906866da663fe90cbb5d5132b0b7b59c546b5c652e36fa9R38

[the remark this answers is not in the document]
Comment (lgrd) [reply .github/workflows/pypi_publish.yml:3]: perfect !

**Review by lgrd — APPROVED** [review]

**Pull request merged**